.env

```
@@ -130,4 +130,5 @@ _APP_PROJECT_REGIONS=default
 _APP_FUNCTIONS_CREATION_ABUSE_LIMIT=5000
 _APP_STATS_USAGE_DUAL_WRITING_DBS=database_db_main
 _APP_TRUSTED_HEADERS=x-forwarded-for
-_APP_POOL_ADAPTER=stack
\ No newline at end of file
+_APP_POOL_ADAPTER=stack
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The _APP_POOL_ADAPTER key should go before the _APP_PROJECT_REGIONS key

(UnorderedKey)